src/compile/common.ts

```diff
@@ -183,3 +186,58 @@ export function timeFormat(model: Model, channel: Channel): string {
   const fieldDef = model.fieldDef(channel);
   return timeFormatExpr(fieldDef.timeUnit, isAbbreviated(model, channel, fieldDef));
 }
+
+export function hasGeoTransform(model: UnitModel): boolean {
+  const geoPathFieldDef = model.encoding().geopath;
+  if (model.mark() === PATHMARK) {
+    if (geoPathFieldDef && geoPathFieldDef.type === GEOJSON) {
+      return true;
+    }
+  }
+  const xFieldDef = model.encoding().x;
+  const yFieldDef = model.encoding().y;
+  if (xFieldDef &&
```

Comment:
add comment that we only require x or y just in case user want to plot lat/long against another measure.  

(So that reader realize this isn't a mistake.)